exchanges/huobi/huobi_test.go

```diff
@@ -1356,9 +1357,8 @@ func TestWSOrderbook(t *testing.T) {
 	assert.Equal(t, 0.56281, liq, "Bid Liquidity should be correct")
 }
 
-// TestWSTradeDetail checks we can send a trade detail through
-// We can't currently easily see the result with the current DB instance, so we just check it doesn't error
-func TestWSTradeDetail(t *testing.T) {
+// TestWsHandleAllTradesMsg checks we can send a trade detail through
```

Comment:
🦟  Nit:
* Concrete wording, particularly around "through" (to where?)
* Swap `checks` for `ensures`; Aiming for consistent usage of `ensures` for regression or specific cases and `exercises` for overall unit testing
```suggestion
// TestWSHandleAllTradesMsg ensures wsHandleAllTrades sends trade.Data to the ws.DataHandler
```

Reply:
changed